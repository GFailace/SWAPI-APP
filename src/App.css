*{
  margin: 0;
  padding: 0;
  outline: 0;
  box-sizing: border-box;
  font-family: 'Star Jedi', sans-serif;
  font-size: 14px;
}

@font-face{
  font-family: Star Jedi;
  src: url("assets/starjedi/Starjedi.ttf");
}

html, body, #root {
  width: 100%;
  height: 100%;
}
body{
    background: #000;
}
.home{
  display: grid;
  grid-template-columns: repeat(3, 1fr);
  grid-gap: 10px;
  margin: 20px 10px;
}