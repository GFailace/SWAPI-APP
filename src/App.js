import React from 'react';
import './App.css';

import Film1 from './components/Film1'
import Film2 from './components/Film2'
import Film3 from './components/Film3';
import Film4 from './components/Film4';
import Film5 from './components/Film5';
import Film6 from './components/Film6';
import Film7 from './components/Film7';
import Header from './layout/Header';

function App() {
  return (
    <div>
      <Header />
      <div className="home">
        <Film1 />
        <Film2 />
        <Film3 />
        <Film4 />
        <Film5 />
        <Film6 />
        <Film7 />
      </div>
    </div>
  );
}

export default App;
