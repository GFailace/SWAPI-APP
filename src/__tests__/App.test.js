import React from 'react'
import {shallow} from 'enzyme'
import App from '../App'

describe('Testando componente principal', () => {
    it('Renderizar conteÃºdo', () => {
        const wrapper = shallow(
            <App />
        )
        expect(wrapper).toMatchSnapshot()
    })
})