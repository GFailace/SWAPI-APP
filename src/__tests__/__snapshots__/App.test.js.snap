// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando componente principal Renderizar conte√∫do 1`] = `ShallowWrapper {}`;
